### PR TITLE
HOSTEDCP-1220: Add multi-arch flag to CLI & associated validation

### PR DESCRIPTION
**Pull request by bryan-cox**
**What this PR does / why we need it**:
Adds the multi-arch flag to the HyperShift CLI to trigger additional validation to ensure a multi-arch release image or stream was used.

**Which issue(s) this PR fixes**:
Fixes [HOSTEDCP-1220](https://issues.redhat.com/browse/HOSTEDCP-1220)

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: cmd/cluster/aws/create_test.go [hunk 1]
```diff
@@ -81,3 +82,41 @@ func Test_ValidateCreateCredentialInfo(t *testing.T) {
 		})
 	}
 }
+
+func Test_validateMultiArchRelease(t *testing.T) {
```
Comment (jparrill) [inline cmd/cluster/aws/create_test.go:86]: For convention s/Test_validateMultiArchRelease/TestValidateMultiArchRelease. I know the upper one is also with _ but I think the rest of tests all uses camel case.

**Review by jparrill — COMMENTED** [review]
small nit, otherwise LGTM

File: cmd/cluster/aws/create_test.go (hunk 1 above)
Comment (bryan-cox) [reply cmd/cluster/aws/create_test.go]: Fixed

**Pull request merged**